_includes/master/endpointport.md

```diff
@@ -0,0 +1,13 @@
+An EndpointPort associates a name with a particular TCP/UDP port of the endpoint, allowing it to
+be referenced as a named port in [policy rules]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy#entityrule).
+
+| Field       | Description                      | Accepted Values   | Schema | Default    |
+|-------------|----------------------------------|-------------------|--------|------------|
+| name        | The name to attach to this port, allowing it to be referred to in [policy rules](./policy#entityrule).  Names must be unique within an endpoint.  | | string |            |
+| protocol    | The protocol of this named port. | tcp, udp          | string |            |
+| port        | The workload port number.        | 1-65535           | int    |            |
+
+> ** NOTE **
+>
+> On their own, EndpointPort entries don't result in any change to the connectivity of the port.
+> They only have an effect if they are referred to in policy.
```

Comment:
Please adapt this according to current master, where the standard Note formatting has slightly changed.  (E.g. look for `alert-info` in `master/getting-started/bare-metal/bare-metal.md`.)